packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -377,6 +387,10 @@ function TldrawEditorWithReadyStore({
 		cameraOptions,
 	})
 
+	const deepLinks = useShallowObjectIdentity(_deepLinks === true ? {} : _deepLinks)
+	const deepLinksRef = useRef(deepLinks)
+	deepLinksRef.current = deepLinks
```

Comment:
any reason we can't chuck this in `editorOptionsRef`? Ideally we wouldn't update refs in render as the render might be aborted with the ref change still taking effect.